docs/quickstart/deploy-stack.md

```diff
@@ -0,0 +1,187 @@
+## Hardware requirements
+
+- A Linux-based OS (e.g., Ubuntu Server 22.04 LTS).
+- At least 8GB RAM with a 2-core CPU.
+- An AMD64 architecture system. 
+
+!!! tip
+    Please follow the [MacOS troubleshooting instructions](mac-troubles.md) to avoid issues that may occur when running the stack on MacOS.
+
+## Prerequisites
+
+1. Install [Kurtosis](https://docs.kurtosis.com/install/).
+
+2. Install the [Foundry toolchain](https://book.getfoundry.sh/getting-started/installation).
+
+3. Run the following script to check you have the remaining required tools and install where missing.
+
+    ```sh
+    curl -s https://raw.githubusercontent.com/0xPolygon/kurtosis-cdk/main/scripts/tool_check.sh | bash
+    ```
+
+## MacOS issues
+
+!!! warning "Resolving MacOS issues"
+    - MacOS set up is a bit flakey.
+    - If you are running MacOS, please follow the [MacOS troubleshooting steps](mac-troubles.md) to help fix issues with the installations, and prior to trying out the RPC calls. 
```

Comment:
Isn't it a bit redundant with the `tip` section? Maybe we could merge those two?

Reply:
100%